### PR TITLE
Fix bug in Twitter browser display

### PR DESCRIPTION
**Pull request by gywgithub**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -217,10 +217,15 @@ <h1 id="dialogTitle">Open File</h1>
...
     <script>
-      const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
-      if (timeZone === 'Asia/Shanghai' && window.location.hostname !== 'cnij.imjoy.io') {
-        document.getElementsByClassName('site-tips')[0].style.display = 'block';
-      }
+      setTimeout(() => {
+        const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+        const hostnameArr = ['cnij.imjoy.io', '0.0.0.0', '127.0.0.1', 'localhost']
+        if (timeZone === 'Asia/Shanghai' && !hostnameArr.includes(window.location.hostname)) {
+          document.getElementsByClassName('site-tips')[0].style.display = 'block';
```
Comment (oeway) [inline src/index.html:224]: To be more efficient, maybe we can assign an id to the site-tips element and use `document.getElementById`

**Comment by oeway** [comment]
There is another place need to be replaced to `document.getElementById` here: https://github.com/imjoy-team/imagej.js/blob/6a0c8e0f770f77c22bc63a00eba6bea3e283b709/src/index.js#L1340

**Comment by gywgithub** [comment]
The CSS code sets display property In the style.css . I think with these settings, we can close the request.
```
.site-tips {
  text-align: center;
  display: none;
}
```

**Comment by oeway** [comment]
Yes, but it might because the css style is in another file which takes time to load.

I think an easy way to fix it completely is to add the style directly to site-tips node.

**Comment by gywgithub** [comment]
OK, I'll improve it again.

**Review by oeway — APPROVED** [review]

**Pull request merged**